inc/agent.class.php

```diff
@@ -1172,13 +1172,6 @@ protected function enrollByInvitationToken($input) {
          return false;
       }
 
-      if (empty($serial) && empty($uuid)) {
```

Comment:
Why did you remove this check if they are mandatory?

Reply:
i tell you in live